lib/src/arg_parser.dart

```diff
@@ -8,6 +8,7 @@
 
 import 'dart:collection';
 
+import '../command_runner.dart';
```

Comment:
It might not be worth pulling this in just for the doc comment reference - I think it might actually pull in a library that otherwise wouldn't be imported if you only import the main library.

Reply:
Fair enough. Removed and referenced the class with backticks instead. It'll be a little harder to navigate to but I'd hope most users can figure out how to find it.